nvflare/job_config/base_app_config.py

```diff
@@ -58,9 +58,6 @@ def add_ext_script(self, ext_script: str):
         if not (os.path.isabs(ext_script) or os.path.exists(ext_script)):
             raise RuntimeError(f"Could not locate external script: {ext_script}")
 
-        if not ext_script.endswith(".py"):
```

Comment:
This change has nothing to do with ScriptExecutor. Please use a different PR.

Reply:
This is minor, I can move to another PR, but this is to support adding more than just .py files to the custom dir (e.g. .toml files in flower's case)